molo/core/views.py

```diff
@@ -10,3 +12,44 @@ def locale_set(request, locale):
 
 def health(request):
     return HttpResponse(status=200)
+
+
+def versions(request):
+    comparison_url = "https://github.com/praekelt/%s/compare/%s...%s"
+    packages = [('molo.core', 'Molo'), ('molo.profiles', 'Profiles'),
+                ('molo.commenting', 'Commenting'), ('molo.polls', 'Polls'),
+                ('molo.yourwords', 'YourWords'), ('molo.servicedirectory',
+                'Srvice Directory')]
```

Comment:
Perhaps lets add a `known_pluggins.py` file somewhere and do `from molo.core.known_plugins import KNOWN_PLUGINS`